internal/tools/pipeline/providers/graph/graph.service.go

```diff
@@ -12,40 +12,62 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package pipelinesvc
+package graph
 
 import (
+	"context"
+	"encoding/json"
+
 	"github.com/sirupsen/logrus"
-	"gopkg.in/yaml.v3"
+	"google.golang.org/protobuf/types/known/structpb"
+	"gopkg.in/yaml.v2"
 
+	"github.com/erda-project/erda-infra/base/logs"
+	basepb "github.com/erda-project/erda-proto-go/core/pipeline/base/pb"
+	"github.com/erda-project/erda-proto-go/core/pipeline/graph/pb"
 	"github.com/erda-project/erda/apistructs"
+	"github.com/erda-project/erda/bundle"
 	"github.com/erda-project/erda/internal/tools/pipeline/services/apierrors"
 	"github.com/erda-project/erda/pkg/i18n"
 	"github.com/erda-project/erda/pkg/parser/pipelineyml"
 	"github.com/erda-project/erda/pkg/strutil"
 )
 
-func (s *PipelineSvc) PipelineYmlGraph(req *apistructs.PipelineYmlParseGraphRequest) (*apistructs.PipelineYml, error) {
```

Comment:
`apistructs.PipelineYmlParseGraphRequest` should be deleted

Reply:
done